kitsune/products/api.py

```diff
@@ -103,8 +27,7 @@ class ProductList(generics.ListAPIView):
 
 
 class TopicShortSerializer(serializers.ModelSerializer):
-    title = LocalizedCharField(source='title',
-                               l10n_context='DB: products.Topic.title')
+    title = LocalizedCharField(l10n_context='DB: products.Topic.title')
```

Comment:
In DRF 3.x, redundant sources are an error. Here it is redundant because DRF infers the source from the name of the assignment on the serializer.